explainer.md

```diff
@@ -557,10 +606,12 @@ interface VRDevice : EventTarget {
 
 dictionary VRSessionCreateParametersInit {
   boolean exclusive = true;
+  VRPresentationContext outputContext = null;
```

Comment:
Do we want to allow this value to be reset?  Or should it only allowed to be set at session creation time?  My inclination is to start as you have it (not resettable) and see if there's use cases that would drive adding the feature.